### ISSUE TITLE
Unable to launch terminal Tabby

### ISSUE THREAD

**Issue by arcreigh** [xpipe-io/xpipe#93]
Tabby Version: 1.0.201 (fresh Install)
Xpipe Version: 1.7.2 (fresh Install)
Windows 11 (fresh install)

```
java.io.IOException: Unable to launch terminal Tabby: Unable to find installation of Tabby.
Maybe try to use a different terminal in the settings.
at io.xpipe.app.util.TerminalHelper.open(TerminalHelper.java:38)
at io.xpipe.ext.base.action.LaunchAction$Action.execute(LaunchAction.java:31)
at io.xpipe.app.comp.store.StoreEntryWrapper.executeDefaultAction(StoreEntryWrapper.java:201)
at io.xpipe.app.comp.store.StoreEntryComp.lambda$createSimple$0(StoreEntryComp.java:101)
at io.xpipe.app.util.ThreadHelper.lambda$runFailableAsync$1(ThreadHelper.java:33)
at java.lang.VirtualThread.run(VirtualThread.java:309)
Caused by: java.io.IOException: Unable to find installation of Tabby
at io.xpipe.app.prefs.ExternalTerminalType$WindowsType.launch(ExternalTerminalType.java:157)
at io.xpipe.app.util.TerminalHelper.open(TerminalHelper.java:35)
... 5 more
```

**Comment by crschnick** [comment]
How is Tabby installed? Did you use the Windows setup or the portable from here https://github.com/Eugeny/tabby/releases/tag/v1.0.201?

For now, you can just use the Custom Terminal setting with `tabby run $CMD`

**Comment by Chloe-ko** [comment]
Just wanted to chime in that I am experiencing the same issue - Tabby installed via the normal windows setup, default location.

**Comment by arcreigh** [comment]
I managed to resolve this by manually adding Tabby to my windows PATH env variable. A bit odd that Tabby didn't do that on install in my opinion.

**Comment by crschnick** [comment]
I think it has something to do with system-wide installations. Currently we only check for per-user installations here: https://github.com/xpipe-io/xpipex/blob/239cc9ee51da6a782b1bdd2088089de72ed5b9ba/app/src/main/java/io/xpipe/app/prefs/ExternalTerminalType.java#L179

I will try to adapt this check for both cases. But yes, adding the executable to the PATH manually will fix this issue for now.

**Comment by crschnick** [comment]
Can you try whether v1.7.3 in https://github.com/xpipe-io/xpipe-ptb works for you?

**Comment by arcreigh** [comment]
I will be happy to test this after I get off work.

**Comment by crschnick** [comment]
This should now be fixed in the latest 1.7.3 release
